README.md

```diff
@@ -17,7 +17,7 @@ This directory contains the following helm charts.
 
 ## Supported Kubernetes versions
 
-* 1.28.x, 1.27.x, 1.26.x, 1.25.x, 1.24.x
+* 1.30.x, 1.29.x, 1.28.x, 1.27.x, 1.26.x
```

Comment:
BTW, can we add a note about how the versions are decided?
The supported versions are the intersection of the supported versions of AKS and EKS at the moment?

Reply:
> BTW, can we add a note about how the versions are decided?

Ah, I see. I think we can add it. I will add it in this PR.

> The supported versions are the intersection of the supported versions of AKS and EKS at the moment?

Yes. Basically, at this time, I decid on our supported versions based on supported versions of EKS and AKS. (Recently, I check GKE and OKE just in case.)

Reply:
@feeblefakie  
I added a description in 3d2994bcfc9ecfb215e40442ad6b54d8dde315e0.

@josh-wong 
We added the new sentence in the README. Could you please take a look at this new sentence? 🙇‍♀️